src/build.sh

```diff
@@ -130,7 +136,7 @@ fi
 
 # Create services
 compose up --detach
-ansible-playbook $ANSIBLE_OPTS ./ansible/playbook_image_service.yml
+ansible-playbook -v $ANSIBLE_OPTS ./ansible/playbook_image_service.yml
```

Comment:
Remove -v, there is ANSIBLE_OPTS env var if you need to set it.

Reply:
done